include/dev/HUDL.hpp

```diff
@@ -137,194 +152,82 @@ class HUDL {
     uint16_t voltages[4];
     uint16_t temperature;
 
-    static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 25;
+    static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 33;
 
     CO_OBJ_T objectDictionary[OBJECT_DICTIONARY_SIZE + 1] = {
-        // Sync ID, defaults to 0x80
-        {
-            CO_KEY(0x1005, 0, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            nullptr,
-            (uintptr_t) 0x80,
-        },
+        MANDATORY_IDENTIFICATION_ENTRIES_1000_1014,
+        HEARTBEAT_PRODUCER_1017(2000),
+        IDENTITY_OBJECT_1018,
+        SDO_CONFIGURATION_1200,
 
         /**
-                 * Information about the hardware , hard coded sample values for now
-                 * 1: Vendor ID
-                 * 2: Product Code
-                 * 3: Revision Number
-                 * 4: Serial Number
-                 */
-        {
-            .Key = CO_KEY(0x1018, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            .Type = nullptr,
-            .Data = (uintptr_t) 0x10,
-        },
-        {
-            .Key = CO_KEY(0x1018, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            .Type = nullptr,
-            .Data = (uintptr_t) 0x11,
-        },
-        {
-            .Key = CO_KEY(0x1018, 3, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            .Type = nullptr,
-            .Data = (uintptr_t) 0x12,
-        },
-        {
-            .Key = CO_KEY(0x1018, 4, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            .Type = nullptr,
-            .Data = (uintptr_t) 0x13,
-        },
+         * *** START RPDO SETTINGS ***
+         */
 
         /**
-                 * SDO CAN message IDS.
-                 * 1: Client -> Server ID, default is 0x600 + NODE_ID
-                 * 2: Server -> Client ID, default is 0x580 + NODE_ID
-                 */
-        {
-            .Key = CO_KEY(0x1200, 1, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            .Type = nullptr,
-            .Data = (uintptr_t) 0x600 + NODE_ID,
-        },
-        {
-            .Key = CO_KEY(0x1200, 2, CO_UNSIGNED32 | CO_OBJ_D__R_),
-            .Type = nullptr,
-            .Data = (uintptr_t) 0x580 + NODE_ID,
-        },
+         * RampUP Board RPDO 0 Settings
```

Comment:
This capitalization is killing me on the inside. I think "Up" or "up" would be okay, but definitely not "UP"